src/main/java/racingcar/service/RaceService.java

```diff
@@ -0,0 +1,39 @@
+package racingcar.service;
+
+import racingcar.domain.Cars;
+import racingcar.domain.Race;
+import racingcar.domain.dto.CarStatusDto;
+import racingcar.domain.dto.RaceResultDto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RaceService {
+
+    private final CarsService carsService;
+
+    public RaceService(final CarsService carsService) {
+        this.carsService = carsService;
+    }
+
+    public Race createRace(final String raceCount) {
+        return Race.createRace(raceCount);
+    }
+
+    public List<RaceResultDto> getRaceResults(final Cars cars, final Race race) {
+        List<RaceResultDto> raceResults = new ArrayList<>();
+        int raceCount = 0;
+
+        while (!race.isRaceFinish(raceCount++)) {
```

Comment:
```suggestion
        while (race.isRunning(raceCount++)) {
```

부정형은 가독성이 떨어져서 이런식으로 네이밍해도 좋을 것 같아요 :)